src/main/java/com/stripe/model/BalanceTransaction.java

```diff
@@ -116,6 +118,7 @@ public void setSource(String source) {
 		this.source = source;
 	}
 
```

Comment:
Could you add a Javadoc comment here? Something like this:

```java
/**
 * @deprecated
 * Recent API versions no longer return this field (https://stripe.com/docs/upgrades#2017-01-27).
 * Prefer listing all transfers with the `transfer_group` parameter: https://stripe.com/docs/api/java#list_transfers-transfer_group.
 */
```

Feel free to reword this as you think is appropriate.